tsconfig.json

```diff
@@ -17,10 +17,11 @@
     "paths": {
       "*": ["./@types/*"]
     },
+    "baseUrl": ".",
```

Comment:
Also curious about this line addition

Reply:
Setting a base url in tsconfig files essentially [tells the compiler where the base or root is for relative path resolution](https://www.typescriptlang.org/tsconfig#baseUrl). So this allows us to use configuration options like  [`paths`](https://www.typescriptlang.org/tsconfig#paths).